blog/members/hellhound3.md

```
@@ -0,0 +1,54 @@
+        <a class="external reference" href="https://github.com/jpchauvel">
+            <iconify-icon icon="simple-icons:github" style="font-size:2em"></iconify-icon>
+        </a>
+    </li>
+</ul>
+```
+
+:Aliases: hellhound
+
+:Ciudad: Lima
+
+:Homepage: https://chauvel.org
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)